geopyspark/tests/base_test_class.py

```diff
@@ -14,7 +14,7 @@
 
 class BaseTestClass(unittest.TestCase):
     if 'TRAVIS' in os.environ:
-        master_str = "local[2]"
+        master_str = "local[1]"
```

Comment:
Why the reduction? This could potentially hide some failures in parallelism.

Reply:
This was one of things I tried to see if it got Travis working. Unfortunately, it did not work. So I think that could be reverted.